FRProximity/FRProximity.xcodeproj/project.pbxproj

```diff
@@ -901,6 +901,9 @@
 				PROVISIONING_PROFILE_SPECIFIER = "";
 				"PROVISIONING_PROFILE_SPECIFIER[sdk=macosx*]" = "";
 				SKIP_INSTALL = YES;
+				SUPPORTED_PLATFORMS = "iphoneos iphonesimulator";
+				SUPPORTS_MACCATALYST = NO;
+				SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD = YES;
```

Comment:
what are those new changes? is it Xcode autogenarted stuff or anything related to passkeys usage?

Reply:
I removed the MacOS Catalyst support. Not that it causes any issues, so far, but it given the wrong idea as our SDKs are not supporting Catalyst